bp_common/src/include/bp_common_aviary_pkg.vh

```diff
@@ -91,9 +90,9 @@ package bp_common_aviary_pkg;
       ,dtlb_els             : 8
 
       ,dcache_sets          : 64
-      ,dcache_assoc         : 8
+      ,dcache_assoc         : 2
```

Comment:
Add new single core configs for the different associativities

Reply:
Followed your suggestion for now and added softcore_l1_small, softcore_l1_medium, single_core_l1_small, single_core_l1_medium. But we do support caches with different associativities. So we definitely need to come up with more scalable names for the different configurations.

Reply:
Good point, can add softcore_l1_hetero : I$ 4-way, D$ 8-way

Reply:
This is a good usecase for https://github.com/black-parrot/black-parrot/issues/422